### DISCUSSION TITLE
puanlarımın görüntülenmesi ve yapmam gerekenler

### DISCUSSION THREAD

**Discussion by erenbasusta** [IBAX-io/go-ibax#616] [Soru Cevap - Türkçe] [answered]
![Screenshot 2022-01-24 at 22-08-53 Welcome to IBAX Network](https://user-images.githubusercontent.com/98027724/150856794-04af256a-3b6f-4a7f-89bd-31c397140dd9.png)

puanlarım bu şekilde görünüyor bunu bir cüzdana vs çekmem gerekiyormu ?

**Comment by Yusufknk** [comment, marked as answer]
Hayır, güncellenen yeni sistemle puan sıralama sistemi oluşturuldu. 1 Şubat'ta cüzdanlara dağıtıma başlanacak.
